targets/mggolangcilint/golangci.yml

```diff
@@ -0,0 +1,263 @@
+    - lll
+
+    # Find slice declarations with non-zero initial length.
+    # [fast: false, auto-fix: false]
+    - makezero
+
+    # Find commonly misspelled English words in comments.
+    # [fast: true, auto-fix: true]
+    - misspell
+
+    # Find naked returns in functions greater than a specified function length.
+    # [fast: true, auto-fix: false]
+    - nakedret
+
+    # Find the code that returns nil even if it checks that the error is not nil.
+    # [fast: false, auto-fix: false]
+    - nilerr
+
+    # Report ill-formed or insufficient nolint directives.
+    # [fast: true, auto-fix: false]
+    - nolintlint
+
+    # Find slice declarations that could potentially be preallocated.
+    # [fast: true, auto-fix: false]
+    - prealloc
+
+    # Find code that shadows one of Go's predeclared identifiers.
+    # [fast: true, auto-fix: false]
+    - predeclared
+
+    # Check Prometheus metrics naming via promlint.
+    # [fast: true, auto-fix: false]
+    - promlinter
+
+    # Check whether Err of rows is checked successfully.
+    # [fast: false, auto-fix: false]
+    - rowserrcheck
+
+    # Check that sql.Rows and sql.Stmt are closed.
+    # [fast: false, auto-fix: false]
+    - sqlclosecheck
+
+    # Go vet on steroids, applying a ton of static analysis checks.
+    # [fast: false, auto-fix: false]
+    - staticcheck
+
+    # Find unused struct fields.
+    # [fast: false, auto-fix: false]
+    - structcheck
+
+    # A replacement for golint.
+    # [fast: false, auto-fix: false]
+    - stylecheck
+
+    # Check the struct tags.
+    # [fast: true, auto-fix: false]
+    - tagliatelle
+
+    # Detect test helpers without t.Helper() call and checks the consistency of test helpers.
+    # [fast: false, auto-fix: false]
+    - thelper
+
+    # Detect inappropriate usage of t.Parallel() method in your Go test codes.
+    # [fast: false, auto-fix: false]
+    - tparallel
+
+    # Parses and type-checks Go code.
+    # [fast: false, auto-fix: false]
+    - typecheck
+
+    # Remove unnecessary type conversions.
+    # [fast: false, auto-fix: false]
+    - unconvert
+
+    # Check Go code for unused constants, variables, functions and types.
+    # [fast: false, auto-fix: false]
+    - unused
+
+    # Find unused global variables and constants.
+    # [fast: false, auto-fix: false]
+    - varcheck
+
+    # Find wasted assignment statements.
+    # [fast: false, auto-fix: false]
+    - wastedassign
+
+    # Detect leading and trailing whitespace.
+    # [fast: true, auto-fix: true]
+    - whitespace
+
+    # Check that no global variables exist.
+    # [fast: true, auto-fix: false]
+    - gochecknoglobals
+
+    # Inspect source code for security problems.
+    # [fast: false, auto-fix: false]
+    - gosec
+
+    # Find sending http request without context.Context.
+    # [fast: false, auto-fix: false]
+    - noctx
+
+    # Detect missing usage of t.Parallel() method in your Go test.
+    # [fast: true, auto-fix: false]
+    - paralleltest
```

Comment:
It is not considered a convention to add t.Parallel in every test I'm Go and some, like hashicorp, even mention avoiding doing it altogether (See https://speakerdeck.com/mitchellh/advanced-testing-with-go?slide=65).

I would rather we don't include this one in the default config

Reply:
Good point! I'm submitting a new PR to remove it

Reply:
https://github.com/einride/mage-tools/pull/58